examples/demo_workflow/app.py

```diff
@@ -32,17 +32,16 @@ def hello_act(ctx: WorkflowActivityContext, input):
     print(f'New counter value is: {counter}!', flush=True)
 
 def main():
-    workflowRuntime = WorkflowRuntime()
-    workflowRuntime.register_workflow(hello_world_wf)
-    workflowRuntime.register_activity(hello_act)
-    workflowRuntime.start()
-
     host = settings.DAPR_RUNTIME_HOST
     if host is None:
         host = "localhost"
     port = settings.DAPR_GRPC_PORT
     if port is None:
         port = "4001"
```

Comment:
I think it's a little strange to set these default values because `settings.DAPR_RUNTIME_HOST` and `settings.DAPR_GRPC_PORT` will never be `None`, right?

Reply:
Correct they will never be none. They load the environment variable or otherwise fall back to the default values.

Do not set those here please!